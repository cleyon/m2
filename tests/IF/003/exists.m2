@if exists(existing-file)
Existing file okay
@else
Existing file failed
@fi
@if exists(non-existing-file)
Failed to detect non-existent file
@else
Non-existent file okay
@fi
