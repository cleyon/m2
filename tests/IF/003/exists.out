Existing file okay
Non-existent file okay
